README.md

```diff
@@ -0,0 +1,354 @@
+Introduction
+------------
+
+Provides a Web server for symbolicating Firefox stacks. It matches PC addresses
+to modules in memory and looks up the corresponding function names in
+server-side symbol files (.SYM files).
+
+This project is intended as a drop-in replacement for
+[Snappy Symbolication Server](https://github.com/mozilla/Snappy-Symbolication-Server/)
+
+Quick Start Guide
+-----------------
+
+1. Install [Python 2.7](https://www.python.org/downloads/) if it is not already
+   installed
+2. Install dependencies:
+   `python -m pip install tornado futures python-memcached psutil`
+3. Install [memcached](http://www.memcached.org/downloads)
+4. Copy/create a configuration file and set values appropriately. See
+   [Configuration File](#configuration-file)
+5. Run `python quickstart.py -c [configuration file]`
+
```

Comment:
This starts servers, which is cool--5 steps to get it going is awesome.

It should probably also document how to stop things.

Something like this:

```
To stop Snappy, run: `python quickstart.py --stop`.

Also, is it likely someone might have memcached running on their system for other reasons already? Is it prudent to run a second one?